packages/core/admin/admin/src/content-manager/components/DynamicTable/BulkActionsBar/index.js

```diff
@@ -218,18 +206,6 @@ const BulkActionsBar = ({
     }
   };
 
-  const handleConfirmPublishAll = async () => {
-    try {
-      setIsConfirmButtonLoading(true);
-      await onConfirmPublishAll(selectedEntries);
-      clearSelectedEntries();
-      setIsConfirmButtonLoading(false);
-    } catch (err) {
-      setIsConfirmButtonLoading(false);
-    }
-    handleToggleShowPublishAllModal();
-  };
-
   // Bulk unpublish
   const handleToggleShowUnpublishAllModal = () => {
```

Comment:
naming feels off, is it `handleToggleShowUnpublishAllModal` or just `toggleUnpublishModal`?